src/web/layouts/CommentLayout.tsx

```diff
@@ -659,7 +659,7 @@ export const CommentLayout = ({
 						currentNavLink={NAV.currentNavLink}
 						palette={palette}
 					/>
-					<Lines count={4} effect="straight" />
+					<Lines count={1} effect="straight" />
```

Comment:
Not sure about this but does having 1 line mean we have 2 lines alongside each other? One from here and one from the top border of the Pillar nav?

Reply:
Hey Oliver - thanks for commenting :) i've checked in with our designer, they'd prefer no line at all, thanks for highlighting this! i'll remove the Lines element completely